genai-perf/genai_perf/subcommand/profile.py

```diff
@@ -86,7 +86,7 @@ def _report_output(
     # Setting to default values to avoid an error.
     elif args.prompt_source == ic.PromptSource.PAYLOAD:
         infer_mode = "request_rate"
-        load_level = "1.0"
+        load_level = "0.0"
```

Comment:
A design artifact of the PA profile data export is that each export requires an inference load mode of either request rate or concurrency and also a particular level for either.

This isn't a very accurate way to describe experiments, and should be redesigned.

In the meantime, GAP needs to access experiment statistics using those values, and in the case of session concurrency mode, it defaults the mode to request rate mode and the level to `0.0`

So this change allows GAP to grab the statistics from session concurrency mode.

It might be better if there's like a version of `get_statistics()` that just returns the first statistics, possibly asserting that there's only one statistic available.

Reply:
The reason why the profile export was putting this metadata along with each experiment is because technically PA can sweep through multiple concurrencies or request rates, and all of them would go in the profile export.

I think the profile export design needs to be revisited, frankly. I don't have all the solutions figured out yet for it.